### PR TITLE
dev-db/mongodb: bump to 4.0.2

### PR DESCRIPTION
**Pull request by hydrapolic**
Tests pass on ~amd64. Thanks :)

````
[resmoke] 2018-09-03T07:49:29.453+0200 Summary of core suite: All 1950 test(s) passed in 207.00 seconds.
1950 test(s) ran in 208.26 seconds (1950 succeeded, 0 were skipped, 0 failed, 0 errored)
    js_tests: All 1950 test(s) passed in 207.00 seconds.
````

### CODE REVIEW

File: dev-db/mongodb/mongodb-4.0.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+	ssl? (
+		!libressl? ( >=dev-libs/openssl-1.0.1g:0= )
+		libressl? ( dev-libs/libressl:0= )
+	)"
+DEPEND="${RDEPEND}
+	${PYTHON_DEPS}
+	dev-python/cheetah[${PYTHON_USEDEP}]
+	dev-python/pyyaml[${PYTHON_USEDEP}]
+	virtual/python-typing[${PYTHON_USEDEP}]
+	dev-util/scons
```
Comment (Zlogene) [inline dev-db/mongodb/mongodb-4.0.2.ebuild]: do you really need this explicitly?

File: dev-db/mongodb/mongodb-4.0.2.ebuild [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+
+SCONS_MIN_VERSION="2.5.0"
+CHECKREQS_DISK_BUILD="2400M"
+CHECKREQS_DISK_USR="512M"
+CHECKREQS_MEMORY="1024M"
+
+inherit check-reqs eapi7-ver flag-o-matic multilib multiprocessing pax-utils python-single-r1 scons-utils systemd toolchain-funcs user
```
Comment (Zlogene) [inline dev-db/mongodb/mongodb-4.0.2.ebuild]: do you trally use multilib eclass?

File: dev-db/mongodb/mongodb-4.0.2.ebuild (hunk 1 above)
Comment (hydrapolic) [reply dev-db/mongodb/mongodb-4.0.2.ebuild]: By checking the eclass, it seems like it's a relic from the past.

File: dev-db/mongodb/mongodb-4.0.2.ebuild (hunk 2 above)
Comment (hydrapolic) [reply dev-db/mongodb/mongodb-4.0.2.ebuild]: This one too, maybe it was needed in the past.

**Pull request closed without merging**